lib/pre-build.js

```diff
@@ -52,6 +52,25 @@ function _getFileName(target) {
     throw new Error('Missing information for naming compiled binary.')
   }
 
+  /**
+   * Electron forks Node and has its own custom ABI versions. Because of this,
+   * the ABI version.included in the binary filename causes issues related to 
```

Comment:
Not gonna hold up PR / go through builds but just noting we have a comment type with a `.` instead of ` ` in here.